packages/utilities/src/utilities/StickyHeader/StickyHeader.js

```diff
@@ -284,12 +284,10 @@ class StickyHeader {
      *   with the elements that should be visible starting at the top of the
      *   viewport.
      */
-    let cumulativeOffset = masthead
-      ? Math.max(
-          Math.min(masthead.offsetTop + oldY - newY, 0),
-          maxScrollaway * -1
-        )
-      : Math.max(Math.min(oldY - newY, 0), maxScrollaway * -1);
+    let cumulativeOffset = Math.max(
+      Math.min((masthead ? masthead.offsetTop : 0) + oldY - newY, 0),
+      maxScrollaway * -1
+    );
```

Comment:
@m4olivei Yeah, previously there were two cases that almost did the exact same thing, so I thought I could simplify.

The difference should be in the `Math.min()` calculation... if the `masthead` exists, we add its `offsetTop` to the `oldY - newY`. Otherwise, we just go with `oldY - newY`. Am _I_ missing something in the logic where that's not the case?

Reply:
Nope! Ends up being the same end result before and after the code change by my read, so looks good 👍